app/common/lib/bookmarkFoldersUtil.js

```diff
@@ -58,7 +58,11 @@ const isFolder = (folder) => {
     return false
   }
 
-  return folder.get('type') === siteTags.BOOKMARK_FOLDER
+  if (folder.get('type') === siteTags.BOOKMARK_FOLDER) {
+    return true
+  }
+  const folderId = folder.get('folderId')
+  return !!(folderId && folderId > 0)
```

Comment:
-1 is a valid folder ID (used for the 'other bookmarks' folder). can this just be `return typeof folderId === 'number'`?